### PR TITLE
SystweakSoftware.DuplicateFilesFixer version 1.2.0.12122

### PR DESCRIPTION
**Pull request by systweaksoftware**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/16544)

### CODE REVIEW

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
I seem to be getting an error while installing the application:
image

image

**Comment by systweaksoftware** [comment]
> I seem to be getting an error while installing the application:
> image
> 
> image

It is not clear what dependencies are missing on your PC. It may be a .NET issue, as .NET version 4.5 is required for installation of Duplicate Files Fixer. Let me know what version you have and I will check that.

File: manifests/s/SystweakSoftware/DuplicateFilesFixer/1.2.0.12122/SystweakSoftware.DuplicateFilesFixer.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: SystweakSoftware.DuplicateFilesFixer
+PackageVersion: 1.2.0.12122
+PackageLocale: en-US
+Publisher: Systweak Software
+PackageName: Duplicate Files Fixer
+License: Systweak Software © 2021
+ShortDescription: Duplicate Files Fixer
```
Comment (ItzLevvie) [inline manifests/s/SystweakSoftware/DuplicateFilesFixer/1.2.0.12122/SystweakSoftware.DuplicateFilesFixer.locale.en-US.yaml:9]:
```suggestion
ShortDescription: Duplicate Files Fixer is a tool designed to find and delete duplicate files effortlessly!
Description: With Duplicate Files Fixer, you can search and delete identical copies of files easily and quickly. It scans duplicate file contents regardless of file name and file format. The preview list gives you the details of duplicate files you can easily do away with.
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
I got it installed:
image

Please see my review.

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**